unstable/wlr-screencontent-unstable-v1.xml

```diff
@@ -0,0 +1,220 @@
+      <arg name="fd" type="fd"
+           summary="fd of the current object"/>
+      <arg name="size" type="uint"
+           summary="size in bytes for the current object"/>
+    </event>
+    <event name="frame_layer">
+      <description summary="layer receiving callback">
+        Callback which serves to supply the client with information on what's
+        contained in each file descriptor and how its laid out.
+        Will be called after the main receive event, once per layer.
+      </description>
+      <arg name="index" type="uint"
+           summary="index of the current plane"/>
+      <arg name="format" type="uint"
+           summary="format of the layer (DRM_FORMAT_*)"/>
+      <arg name="num_planes" type="uint"
+           summary="number of planes in the current layer (max 4)"/>
+    </event>
+    <event name="frame_plane">
+      <description summary="plane info receiving callback">
+        Callback which supplies the client with plane information for each
+        layer.
+      </description>
+      <arg name="index" type="uint"
+           summary="index of the current plane"/>
+      <arg name="layer_index" type="uint"
+           summary="index of the current layer"/>
+      <arg name="object_index" type="uint"
+           summary="index of the object the plane applies to"/>
+      <arg name="offset" type="uint"
+           summary="starting point for the data in the object's fd"/>
+      <arg name="stride" type="uint"
+           summary="line size in bytes"/>
+    </event>
+    <event name="frame_ready">
+      <description summary="indicates frame is available for reading">
+        Called as soon as the frame is presented, indicating it is available
+        for reading.
+        The timestamp is expressed as tv_sec_hi, tv_sec_lo, tv_nsec triples,
+        each component being an unsigned 32-bit value. Whole seconds are in
+        tv_sec which is a 64-bit value combined from tv_sec_hi and tv_sec_lo,
+        and the additional fractional part in tv_nsec as nanoseconds. Hence,
+        for valid timestamps tv_nsec must be in [0, 999999999].
+      </description>
+      <arg name="tv_sec_hi" type="uint"
+           summary="high 32 bits of the seconds part of the timestamp"/>
+      <arg name="tv_sec_lo" type="uint"
+           summary="low 32 bits of the seconds part of the timestamp"/>
+      <arg name="tv_nsec" type="uint"
+           summary="nanoseconds part of the timestamp"/>
+    </event>
+    <event name="frame_abort">
+      <description summary="indicates the frame is no longer valid">
+        If the frame is no longer valid after the "receive" event has been
+        called, this callback will be used to inform the client to scrap the
+        frame. The client can register to receive another frame callback here
+        in case capturing becomes possible again.
+      </description>
+    </event>
+
+    <request name="frame_destroy" type="destructor">
```

Comment:
Since thiese events and requests are in the `zwlr_screencontent_frame_v1` interface, no need to add a "frame_" prefix to those, otherwise we'll end up with symbols with "frame" appearing twice (like `zwlr_screencontent_frame_v1_frame_destroy(struct zwlr_screencontent_frame_v1 *frame)`).